site/sfguides/src/intro_to_snowpark_container_services/intro_to_snowpark_container_services.md

```diff
@@ -0,0 +1,544 @@
+
+The introduction of SPCS on Snowflake includes the incorporation of new object types and constructs to the Snowflake platform, namely: images, image registry, image repositories, compute pools, specification files, services, and jobs.
+
+For more information on these objects, check out [this article](https://medium.com/snowflake/snowpark-container-services-a-tech-primer-99ff2ca8e741) along with the Snowpark Container Services [documentation]().
+
+### What you will learn 
+- The basic mechanics of how Snowpark Container Services works
+- How to deploy a long-running service with a UI
+- How to deploy a Service Function to perform basic calculations
+
+### Prerequisites
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed
+    >**Download the git repo here: https://github.com/sfc-gh-cbaechtold/spcs-101-quickstart.git**
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed
+- [Python 3.10](https://www.python.org/downloads/) installed
+    - Note that you will be creating a Python environment with 3.10 in the **Setup the Local Environment** step
+- (Optional) [VSCode](https://code.visualstudio.com/) (recommended) with the [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker), [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python), and [Snowflake](https://marketplace.visualstudio.com/items?itemName=snowflake.snowflake-vsc) extensions installed.
+- A Snowflake account. If you do not have a Snowflake account, you can register for a [free trial account](https://signup.snowflake.com/).
+- A Snowflake account login with a role that has the `ACCOUNTADMIN` role. If not, you will need to register for a free trial or work with your `ACCOUNTADMIN` to perform the account setup.
+
+### What You’ll Build 
+- A hosted Jupyter Notebook service running inside of Snowpark Container Services
+- A Python REST API to perform basic temperature conversions
+- A Snowflake Service Function that leverages the REST API
+
+<!-- ![e2e_ml_workflow](assets/snowpark-ml-e2e.png) -->
+
+<!-- ------------------------ -->
+## Set up the Snowflake environment
+Duration: 5
+
+Run the following SQL commands in [`00_setup.sql`](https://github.com/sfc-gh-cbaechtold/spcs-101-quickstart/blob/dev/00_setup.sql) using the Snowflake VSCode Extension OR in a SQL worksheet to create the role, database, warehouse, and stage that we need to get started:
+```SQL
+// Create an SPCS_USER_ROLE with required privileges
+USE ROLE ACCOUNTADMIN;
+CREATE ROLE SPCS_USER_ROLE;
+GRANT CREATE DATABASE ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE WAREHOUSE ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE COMPUTE POOL ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE INTEGRATION ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT MONITOR USAGE ON ACCOUNT TO  ROLE  SPCS_USER_ROLE;
+GRANT IMPORTED PRIVILEGES ON DATABASE snowflake TO ROLE SPCS_USER_ROLE;
+
```

Comment:
I believe there is now a new permissions that allows a user to create services with public endpoints.